pages/advanced-algorithms/available-algorithms/algo.mdx

```diff
@@ -0,0 +1,442 @@
+There are three possible ways to define a relationship inside the filter. Firstly: `<REL_NAME>`, means only incoming relationships with the
+provided name will be considered by the algorithm. `REL_NAME>` means only outgoing relationships with the provided name will be considered
+by the algorithm. Lastly, `REL_NAME` means that bout outgoing and incoming relationships with the provided name will be considered
+by the algorithm.
+- `duration: bool, default = false` - if `true`, means that the distance and heuristic are of a `Duration` data type. NOTE: If they are not created
+as a `Duration` data type, and the `duration` config is specified as `true`, an exception is thrown. Also, if this config is `true`, the `weight`
+output will be returned as `int`, representing the duration in microseconds.
+- `unweighted: bool, default = false` - if `true`, the algorithm will run unweighted, using only heuristics, not distances between nodes.
+This config will still return the path weight, but it will count each distance the same, as 10.
+- `epsilon: double, default = 1.0` - if the user has provided an admissible heuristic, or if they use the default heuristic,
+they can choose to use the Bounded relaxation property of A *, and multiply the heuristic by `epsilon` value. This will result in a faster algorithm, with a solution
+at most `epsilon` times worse than the base algorithm. See [more](https://en.wikipedia.org/wiki/A*_search_algorithm#Bounded_relaxation).
+
+
+
+{<h4> Usage: </h4>}
+
+{<h5> Graph creation: </h5>}
+
+
+Graph will have five `City` nodes, and four `MO` nodes. Each node will have a defined latitude and longitude.
+Relationships will be of types `ROAD` and `RIVER`. Each relationship has a defined distance property.
+
+```cypher
+CREATE (c1:City1 {lat: -1, lon: -1}), (c2:City2 {lat: 1, lon: -11}),
+(c3:City3 {lat: 1, lon: 1}), (c4:City4 {lat: -1, lon: 1}), (c5:City5 {lat: 0, lon: 0}),
+(m1:MO1 {lat: -0.5, lon: -0.5}), (m2:MO2 {lat: 0.5, lon: -0.5}),
+(m3:MO3 {lat: 0.5, lon: 0.5}) ,(m4:MO4 {lat: -0.5, lon: 0.5}),
+(c1)-[r1:ROAD {distance: 300}]->(c2),
+(c2)-[r2:ROAD {distance: 300}]->(c3),
+(c4)-[r3:ROAD {distance: 280}]->(c3),
+(c4)-[r4:ROAD {distance: 310}]->(c1),
+(c1)-[r5:RIVER {distance: 90}]->(m1),
+(c2)-[r6:ROAD {distance: 90}]->(m2),
+(c3)-[r7:ROAD {distance: 85}]->(m3),
+(c4)-[r8:ROAD {distance: 100}]->(m4),
+(m1)-[r9:RIVER {distance: 140}]->(m4),
+(m4)-[r10:RIVER {distance: 145}]->(m3),
+(m3)-[r11:RIVER {distance: 130}]->(m2),
+(m2)-[r12:RIVER {distance: 125}]->(m1),
+(m2)-[r13:ROAD {distance: 90}]->(c5),
+(m3)-[r14:RIVER {distance: 100}]->(c5)
+```
+![](/pages/advanced-algorithms/available-algorithms/algo/a_star_graph.png)
+
+{<h5> Default usage: </h5>}
+
+Since the distance property is already named `distance`, and latitude and longitude properties are already named `lat` and `lon`,
+config map is unnecessary, and the algorithm can be run without it.
```

Comment:
```suggestion
Since the distance property is named `distance`, and latitude and longitude properties are named `lat` and `lon`, a config map is unnecessary, and the algorithm can be run without it.
```